timesketch/frontend-ng/src/components/Explore/UnfurlDialog.vue

```diff
@@ -12,66 +12,61 @@ limitations under the License.
 -->
 <template>
   <v-card>
-    <v-card-title>
-      <v-img :src="getUnfurlLogo" max-height="32" contain class="mb-2"></v-img>
-    </v-card-title>
+    <v-toolbar flat color="transparent">
+      <v-img position="left" :src="getUnfurlLogo" max-height="24" contain class="ml-2"></v-img>
+    </v-toolbar>
+
     <v-card-subtitle class="pt-1">
-      <span><b>Input:</b> {{ url }}</span>
+      <div class="mb-2"><b>Input:</b> {{ url }}</div>
+      <div v-if="nodeContext">
+        <b>Selected node info: </b>
+        <span v-html="sanitizeHtml(nodeContext)"></span>
+      </div>
+      <div v-else>Select a node in the graph below to get more information.</div>
     </v-card-subtitle>
+
     <v-card-text>
       <div v-show="!unfurlReady">
         <v-progress-linear color="primary" indeterminate> </v-progress-linear>
       </div>
-      <v-divider></v-divider>
-      <v-container fluid v-show="unfurlReady">
-        <!-- Cytoscape container -->
-        <div style="font-size: medium" class="py-1 px-1">
-          <span style="font-weight: bold">Selected node context: </span>
-          <br />
-          <span style="font-style: italic" v-html="nodeContext"></span>
+
+      <v-toolbar dense flat color="transparent" class="mt-n8">
```

Comment:
```suggestion
      <v-toolbar v-show="unfurlReady" dense flat color="transparent" class="mt-n8">
```

because: 
![image](https://github.com/google/timesketch/assets/99879757/60898a11-3e23-45db-bc01-4119023b2809)